requirements.txt

```diff
@@ -1,2 +1,2 @@
-cython==0.25.1
-ql-cystatsd==1.1.0
+cython>=0.25.1
+git+https://github.com/qntln/cystatsd.git#egg=ql-cystatsd
```

Comment:
I have had a lot of painful experiences with open-ended dependencies like these. Sooner or later a version will come out that's not backwards compatible and new builds suddenly start failing.

I am up for updating `cython` to the newest current version, but pinning it with `==`. Since the major version is still `0` it is possible that future minor versions will break compatibility according to semver rules.

`ql-cystatsd` is our fork of https://github.com/scivey/cystatsd that will hopefully cease to exist once some PRs are merged upstream. So I can't make any guarantees about its future versions either.